src/table/table-head.component.ts

```diff
@@ -0,0 +1,242 @@
+import {
+	ApplicationRef,
+	Component,
+	HostBinding,
+	Input,
+	Output,
+	EventEmitter
+} from "@angular/core";
+
+import { TableModel } from "./table.module";
+import { I18n } from "./../i18n/i18n.module";
+
+@Component({
```

Comment:
Add component description that explains where and how it is used.
This is especially important here since we also have a variety of `table-header*` components and this will easily create confusion among users.